Fails with python 3.6

Fails when attempting to build with python 3.6:
```
Collecting enum34==1.1.2 (from -r requirements.txt (line 3))
  Using cached enum34-1.1.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Users/jgregory/.virtualenvs/sdx-decrypt/lib/python3.6/site-packages/setuptools/__init__.py", line 5, in <module>
        import distutils.core
      File "/Users/jgregory/.virtualenvs/sdx-decrypt/lib/python3.6/distutils/__init__.py", line 4, in <module>
        import imp
      File "/Users/jgregory/.virtualenvs/sdx-decrypt/lib/python3.6/imp.py", line 27, in <module>
        import tokenize
      File "/Users/jgregory/.virtualenvs/sdx-decrypt/lib/python3.6/tokenize.py", line 33, in <module>
        import re
      File "/Users/jgregory/.virtualenvs/sdx-decrypt/lib/python3.6/re.py", line 142, in <module>
        class RegexFlag(enum.IntFlag):
    AttributeError: module 'enum' has no attribute 'IntFlag'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/3x/cbmnscz143q40msktt0r68cr0000gn/T/pip-build-pi000xiz/enum34/
make: *** [build] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,6 @@
 language: python
 python:
+    - "3.6"    
     - "3.5"
     - "3.4"
 install:

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+### Unreleased
+  - Remove enum34 from requirements.txt
+
 ### 1.3.0 2017-07-25
   - Change all instances of ADD to COPY in Dockerfile
   - Remove use of SDX_HOME variable in makefile

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,5 @@
 cffi==1.9.1
 cryptography==1.7.1
-enum34==1.1.2
 Flask==0.10.1
 gunicorn==19.6.0
 idna==2.0
@@ -17,3 +16,4 @@ sdx-common==0.7.0
 six==1.10.0
 structlog==16.1.0
 Werkzeug==0.11.9
+

